### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by anirudhmalik** [anirudhmalik/xhunter#27] [Ideas]
- slack notification when victim uses payload/malicious app. so you don't have to `listen on connection` for ever.
- inject permissions in payload.

**Comment by Storm8lack** [comment]
> * notificación de holgura cuando la víctima usa carga útil/aplicación maliciosa. para que no tengas que hacerlo `listen on connection`para siempre.
> * inyectar permisos en la carga útil.

Hi friend!!!  Another important feature is the GPS location of the victim, that interests me a lot.

**Comment by HeatWavve** [comment]
@anirudhmalik 

1. Please add slack webhook for notification listening
2. Location tracking
3.  Screen monitoring.

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
